docs/topics/email.txt

```diff
@@ -399,9 +399,27 @@ Django's email library, you can do this using the
 
     A subclass of :class:`~django.core.mail.EmailMessage` that allows
     additional versions of the message body in the email via the
-    ``attach_alternative()`` method. This directly inherits all methods
-    (including the class initialization) from
-    :class:`~django.core.mail.EmailMessage`.
+    :meth:`~django.core.mail.EmailMultiAlternatives.attach_alternative` method.
+    This directly inherits all methods (including the class initialization)
+    from :class:`~django.core.mail.EmailMessage`.
+
+    .. attribute:: alternatives
+
+        A list of named tuples with attributes ``(content, mimetype)``. This is
+        particularly useful in tests::
+
+            self.assertEqual(len(msg.alternatives), 1)
+            self.assertEqual(msg.alternatives[0].content, html_content)
+            self.assertEqual(msg.alternatives[0].mimetype, "text/html")
+
+        Alternatives should only be added using the
+        :meth:`~django.core.mail.EmailMultiAlternatives.attach_alternative`
```

Comment:
As far as I'm aware, it should work without a full path as it's defined here :thinking: 
```suggestion
        :meth:`attach_alternative`
```

Reply:
🤯 TIL